Makefile

```diff
@@ -1,11 +1,5 @@
 check_dirs := score_analysis tests
 
-# This target runs checks on all files and potentially modifies some of them
-style:
-	poetry run black $(check_dirs)
-	poetry run isort $(check_dirs)
-	poetry run flake8 $(check_dirs)
-
```

Comment:
We should also remove `--black --isort` from the test command below and replace it with an explicit `ruff` check via Github Actions (maybe in a follow up PR). And then remove `pytest-black`, etc. from the python environment.

Reply:
Removed `--black --isort` for now.